src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.tsx

```diff
@@ -24,6 +26,7 @@ const KeyStatTile = ({
       className={classNames(styles.tile, {
         [styles.reordering]: isReordering,
       })}
+      data-testid={`${testId}-dataBlockId-${dataBlockId}`}
```

Comment:
Not too keen on passing `dataBlockId` through like this. Key stat tiles aren't data blocks, so this test id seems strange.

There should be alternative selectors to access key stat tiles via some parent element in most cases e.g. `#releaseHeadlines-summary [data-testid="keyStat"]` for key stats in the headlines.